### DISCUSSION TITLE
BuilderWandUseEvent API

### DISCUSSION THREAD

**Discussion by nivcoo** [BG-Software-LLC/WildTools#104] [Pending Feature Requests]
Hello, can you add origin block in API ? To get type of all block because actually we have only list of location and we have to getTargetBlock of player to know the block.

**Comment by OmerBenGera** [comment]
The list of locations are the blocks that were broken.

**Reply by nivcoo** [reply]
Broken ? With Builder wand hum, i think no

**Reply by OmerBenGera** [reply]
> Broken ? With Builder wand hum, i think no

Then list of blocks that were placed.

**Reply by nivcoo** [reply]
Yes so how can we get the type of block ?

**Reply by OmerBenGera** [reply]
> Yes so how can we get the type of block ?

Location#getBlock

**Reply by nivcoo** [reply]
Yes but no, it's reply air because it's before the placement of blocks
